pymws/exceptions.py

```diff
@@ -24,3 +24,14 @@ class AccessDenied(MWSError):
 class SignatureDoesNotMatch(MWSError):
     """The signature used does not match the server's
     calculated signature value."""
+    pass
+
+
+class QuotaExceeded(MWSError):
+    """The total number of requests in an hour was exceeded."""
+    pass
+
+
+class RequestThrottled(MWSError):
```

Comment:
this should also provide the retry-after or retry-on details, right?
https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Throttling.html

```x-mws-quota-max: 3600
x-mws-quota-remaining: 10
x-mws-quota-resetsOn: Wed, 06 Mar 2013 19:07:58 GMT
```